### ISSUE TITLE
format on paste deletes code

### ISSUE THREAD

**Issue by anandijain** [julia-vscode/julia-vscode#3125]
i couldn't seem to find this issue on here, but i know it's been referenced on slack frequently.

i like format on paste, but i just default to command-z after every paste, since it almost always deletes some of my code.

is there plans to fix this?

**Comment by davidanthoff** [comment]
Agreed, we should really fix this rather sooner than later, and if that (for the short term) means disabling features then that might have to be the solution. But corruption of user code really is something we should just try to mitigate as quickly as possible.

**Comment by anandijain** [comment]
if there's no plan to get it fixed, can we get a patch that disables this by default. it is a showstopper

**Comment by davidanthoff** [comment]
Closing as a duplicate of #3008 — same format-on-paste data loss. Please follow that issue for updates.
